tests/lib/rules/sort-prop-types.js

```diff
@@ -565,6 +565,42 @@ ruleTester.run('sort-prop-types', rule, {
     }]
   }, {
     code: [
+      'const First = (props) => <div />;',
+      'const propTypes = {',
```

Comment:
Hopefully nothing; I'm just making sure that an export declaration works just as well as a variable declaration :-)